src/main/java/com/denizenscript/depenizen/bukkit/properties/crackshot/CrackShotEntityProperties.java

```diff
@@ -0,0 +1,85 @@
+package com.denizenscript.depenizen.bukkit.properties.crackshot;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.properties.Property;
+import com.denizenscript.denizencore.tags.Attribute;
+import com.denizenscript.depenizen.bukkit.bridges.CrackShotBridge;
+import org.bukkit.entity.Projectile;
+import org.bukkit.entity.TNTPrimed;
+
+public class CrackShotEntityProperties implements Property {
+
+    public static final String[] handledTags = new String[]{
+            "shot_from"
+    };
+
+    EntityTag entity;
+
+    private CrackShotEntityProperties(EntityTag entity) {
+        this.entity = entity;
+    }
+
+    public static CrackShotEntityProperties getFrom(ObjectTag object) {
+        if (!describes(object)) {
+            return null;
+        }
+        else {
+            return new CrackShotEntityProperties((EntityTag) object);
+        }
+    }
+
+    public static boolean describes(ObjectTag object) {
+        return object instanceof EntityTag;
+    }
+
+    @Override
+    public String getPropertyString() {
+        return null;
+    }
+
+    @Override
+    public String getPropertyId() {
+        return "CrackShotEntity";
+    }
+
+    @Override
+    public String getAttribute(Attribute attribute) {
+        if (attribute == null) {
+            return null;
+        }
+
+        // <--[tag]
+        // @attribute <EntityTag.shot_from>
```

Comment:
Change this to something like `crackshot_weapon_name`. `shot_from` is too generic.

Reply:
Done.